app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,46 @@
+// Place all the styles related to the Profiles controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+/* General */
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  background-image: url('http://s3.amazonaws.com/codecademy-content/courses/learn-rails/img/bg-static.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+.container {
+  margin: 0 auto;
+  width: 960px;
+}
+
+/* Main */
+.main {
+  color: #fff;
+  position: relative;
+  top: 100px;
+  text-align: center;
+}
+
+.main h1 {
+  font-size: 80px;
+  font-weight: 300;
+}
```

Comment:
The styelsheet this is inside is specific to profile.  Since main is just a div surrounding the entire profile page, you really don't need to specify this as main.  h1 is enough.  
Also it is better practice to use nesting for styles.  For example:
```
.main {
  color: #fff;
  position: relative;
  top: 100px;
  text-align: center;
  h1 {
    font-size: 80px;
    font-weight: 300;
  }
}
```